### DISCUSSION TITLE
my request got apporved but not receive benifits

### DISCUSSION THREAD

**Discussion by kush687** [community/community#145970] [GitHub Education]
### Hi everyone,

Hi everyone,
It’s been more than 3 days since my approval, and I still haven’t received the Student Developer Pack.
How much longer should I wait?

**Comment by quoc10973** [comment]
Keep waiting, maybe more 3 or 4 days for received pack. Your request could be rejected if your ID card is blur or low quality.

**Comment by GabJuarez** [comment]
Hi!

It usually takes up to a week for the benefits of the Student Developer Pack to be activated after approval. If it has been more than 7 days and you still haven’t received them, I recommend contacting GitHub Education Support directly.

[GitHub Support](https://support.github.com/contact/education)

Hope this helps!😸

Let me know if you’d like to tweak it further. (Mark this as an answer please)

**Comment by BhathiyaPrasad** [comment]
It usually takes like 3 days i also waited for 3 days to receive the benefits stay calm brother you will receive your benefits soon

**Comment by davevad93** [comment]
Hi @kush687 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @kush687  was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
